assets/xml/objects/object_skb.xml

```diff
@@ -0,0 +1,64 @@
+<Root>
+    <File Name="object_skb" Segment="6">
+        <!-- Stalchild skeleton -->
+        <Skeleton Name="gStalchildSkel" Type="Normal" LimbType="Standard" Offset="0x41F8"/>
+
+        <!-- Stalchild Limbs -->
+        <Limb Name="gStalchildRootLimb" Type="Standard" Offset="0x40C8"/>
+        <Limb Name="gStalchildWaistLimb" Type="Standard" Offset="0x40D4"/>
+        <Limb Name="gStalchildRoot2Limb" Type="Standard" Offset="0x40E0"/>
```

Comment:
I wonder if this should stay unnamed, since it's not completely obvious what it's for.